airbyte-integrations/connectors/source-faker/integration_tests/expected_records.txt

```diff
@@ -0,0 +1,10 @@
+{"type": "RECORD", "record": {"stream": "Users", "data": {"job": "Musician", "company": "Williams-Sheppard", "ssn": "498-52-4970", "residence": "Unit 5938 Box 2421\nDPO AP 33335", "current_location": [52.958961, 143.143712], "blood_group": "B+", "website": ["http://www.rivera.com/", "http://grimes-green.net/", "http://www.larsen.com/"], "username": "leeashley", "name": "Gary Cross", "sex": "M", "address": "711 Golden Overpass\nWest Andreaville, MA 71317", "mail": "tamaramorrison@hotmail.com", "birthdate": "1945-06-05", "id": 1, "created_at": "2022-04-12T17:32:21.905904", "updated_at": "2022-04-12T17:32:21.905904"}, "emitted_at": 1649809941000}}
```

Comment:
Using a fixed seed, we can confirm the records produced in tests.
However, I've modified this file locally, and I don't see failures from the SAT.  I may be using the `expect_records` directive in `acceptance-test-config.yml` wrong